distributed/cli/tests/test_dask_submit.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import print_function, division, absolute_import
+from distributed.utils_test import valid_python_script, popen, invalid_python_script, executor_contract_script
+
+
+def test_dask_submit_cli(valid_python_script, invalid_python_script):
+    with popen(['dask-scheduler', '--port', '8989']) as sched:
+        with popen(['dask-remote']) as remote:
+            with popen(['dask-submit', '127.0.0.1:8788', str(valid_python_script)]) as submit:
+                for x in range(5):
+                    line = submit.stdout.readline()
+                    if b'hello world!\\n' in line:
+                        break
+
+            with popen(['dask-submit', '127.0.0.1:8788', str(invalid_python_script)]) as submit:
+                for x in range(5):
+                    line = submit.stdout.readline()
+                    if b'Traceback' in line:
+                        break
```

Comment:
I am generally not happy how these tests are run. I tried a few ways to get `assert` statements here but i couldnt make travis work with them well. I will continue thinking about this. It seems like a general enough test type that it will help to have it in a nicer state.